src/components/views/Registration.tsx

```diff
@@ -42,16 +53,16 @@ const Registration = () => {
   const doRegister = async () => {
     try {
       const requestBody = JSON.stringify({ username, name, password });
-      const response = await api.post("/users", requestBody);
+      const response = await api.post("/register", requestBody);
```

Comment:
@B1s9l  error handling. depending on return status, give different hint to user. not just "Something went wrong"